airbyte-integrations/connectors/destination-mysql-strict-encrypt/build.gradle

```
@@ -25,3 +25,10 @@ dependencies {
     implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
     integrationTestJavaImplementation files(project(':airbyte-integrations:bases:base-normalization').airbyteDocker.outputs)
 }
+
+tasks.named("airbyteDocker") {
```

```suggestion
tasks.named("airbyteDockerMySql") {
```

(which actually solves the comment on the next line, because now it's only building one image!)